.github/workflows/verify.yml

```
@@ -237,6 +237,9 @@ jobs:
   test-publishing:
     name: Build and publish Python 🐍 distributions 📦 to TestPyPI
     runs-on: ubuntu-latest
+    environment: testpypi
+    permissions:
+        id-token: write
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)